src/com/xilinx/rapidwright/edif/EDIFTools.java

```diff
@@ -1736,7 +1736,7 @@ public static EDIFNetlist createFlatNetlist(EDIFNetlist netlist, String partName
         for (EDIFPort topPort : netlist.getTopCell().getPorts()) {
             EDIFPort flatPort = flatTop.createPort(topPort);
             if (flatPort.isBus()) {
-                int[] indicies = flatPort.getBitBlastedIndicies();
+                int[] indicies = flatPort.getBitBlastedIndices();
```

Comment:
Misspelled here and below too.

Reply:
Thanks, I saw it and I think I have fixed all instances of `indicies` -> `indices`.